src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/VersionControllerAT.java

```diff
@@ -0,0 +1,74 @@
+package de.zalando.aruha.nakadi.webservice;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import static com.jayway.restassured.RestAssured.given;
+import de.zalando.aruha.nakadi.config.JsonConfig;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.Map;
+import org.apache.http.HttpStatus;
+import static org.hamcrest.CoreMatchers.is;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.junit.Test;
+import org.springframework.util.StringUtils;
+
+public class VersionControllerAT extends BaseAT {
+    private static final String VERSION_RESOURCE = "/version";
+    private static final ObjectMapper mapper = (new JsonConfig()).jacksonObjectMapper();
+
+    @Test(timeout = 10000)
+    public void scmSourceShouldContainCorrectData() {
+        given().when().get(VERSION_RESOURCE).then().statusCode(HttpStatus.SC_OK).body(is(new Matcher<Object>() {
```

Comment:
I didn't expected to have a custom matcher for these assertions. What it does it to check if the json structure matches the expected one, right? Maybe we could achieve it with some already existing json matcher.